iidm/iidm-converter-api/src/main/java/com/powsybl/iidm/export/Exporter.java

```diff
@@ -43,4 +44,16 @@
      */
     void export(Network network, Properties parameters, DataSource dataSource);
 
+    /**
+     * Export a model.
+     *
+     * @param network the model
+     * @param parameters some properties to configure the export
+     * @param dataStore data store
+     * @param exported file name
+     */
+    default void export(Network network, Properties parameters, DataStore dataStore, String filename) {
+        throw new UnsupportedOperationException("Method not implemented");
```

Comment:
Is it easy to convert the dataStore to a dataSource to avoid a breaking change?

Reply:
@quinarygio : do you think it's possible to implement this as a `DataSource` adapter (wrapper) around a delegate `DataStore` ?

Not 100% sure it's possible, but it would avoid to enforce `DataStore` implementations to implement a bothering legacy `toDataStore()` method.